server/schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE users (
+  id                    SERIAL PRIMARY KEY,
+  firstname             VARCHAR(30) NOT NULL,
+  lastname              VARCHAR(30) NOT NULL,
+  email                 VARCHAR(30) NOT NULL,
+  password              VARCHAR(12) NOT NULL,
+  createdAt             TIMESTAMP  NOT NULL
+);
+
+CREATE TABLE stars (
+    id                  SERIAL PRIMARY KEY,
+    user_id             INTEGER,
+    competence_id       INTEGER NOT NULL,
+    title               TEXT NOT NULL,
+    situation           TEXT NOT NULL,
+    task                TEXT NOT NULL,
+    action              TEXT NOT NULL,
+    result              TEXT NOT NULL,
+    createdAt           TIMESTAMP  NOT NULL,
+    UpdatedAt           TIMESTAMP  NOT NULL
+);
+
+CREATE TABLE competencies (
+    id                  SERIAL PRIMARY KEY,
```

Comment:
I think, where we have a 2 column table where 1 column is the ID we can probably collapse this. What I mean by this is that we can probably add the column `name` as a column in the `STARS` table instead of in a different table all together

Reply:
The competencies table is contain the group of interview question. on the Wireframe is have it own table. Thus I think it needs its own table.